.github/workflows/main.yaml

```diff
@@ -28,95 +28,95 @@ jobs:
     - name: Run Tox
       run: tox -e lint
 
-  py36:
+  py37:
```

Comment:
Python 3.7 is deprecated

Reply:
Yeah :100: but I think it should be a separate PR to update all the dependencies